RELEASE.md

```diff
@@ -167,7 +167,11 @@ Promoting the images basically means that images, that were pushed to staging co
     - annotations
       - artifacthub.io/prerelease: "true" 
       - artifacthub.io/changes: |
-        - Add the titles of the PRs merged after previous release here. I used the github-cli to get that list like so `gh pr list -s merged -L 38 -B main | cut -f1,2`
+        - Add the titles of the PRs merged after previous release here. github-cli was used to get a list of PRs like so 
```

Comment:
when you say `here` what that means? where we should add this? 
the second part can be something like `The tool `github-cli` cab be used to get a list of PRs`

Reply:
I wanted to ask because I just had to rebase it - why is this better than
```bash
git log <prev-tag>..main --pretty=%s
```
??

The list generates like 
```
fix custom-error-pages file not exist (#7728)
Bump google.golang.org/grpc from 1.41.0 to 1.43.0 (#8054)
Remove force sync / 30s for non-external backends (#7958)
Report expired certificates (#8045) (#8046)
docs: fix typo'd executible name (#8041)
Bump github.com/spf13/cobra from 1.2.1 to 1.3.0 (#8058)
Add stream-snippet as a ConfigMap and Annotation option (#8029)
align bug report with feature request regarding kind documentation (#8051)
Disable default modsecurity_rules_file if modsecurity-snippet is specified (#8021)
```

and the #nnnn is automatically expanded to a link when viewing on github

Reply:
> when you say `here` what that means? where we should add this? the second part can be something like
> 
> ```
> The tool `github-cli` cab be used to get a list of PRs
> ```

@cpanato Apologies, I missed editing this part. I making the changes now. Need to get elaborate on generating the Changelog type PRs list here as well.

@afirth, thanks for your comment. I need to do a lot of explaining here.
- One reason was I was/am favoring gh-cli. I will change to your suggestion of using the git executable, after I have done some background work and find it better suited. The factors influencing the decisions are ;
  - There are 2 places we need to present the Changelog. So, at the back of my mind, I wanted to have one method to generate the Changelog, for both destinations. One place is the Changelog in github, where I realize only now, that  the `#nnnn` will expand to a link. Thanks for helping me realize that. But there is another Changelog that is included with the helm-chart, that goes on artifacthub.io. The `#nnnn` almost certainly would not expand to a link.
  - Another reason is that the git command output has the PR number at the end of each line after the PR-Title. But all past versions of our Changelog have the PR number, at the beginning of the line. It seems some trivial automation or output format flags to the `git log` command can produce desired output format. Just that I don't know. I will need to read and learn how to do that.
  - Eventually I am hoping `github cli` can produce the Changelog format we need, both for github and artifacthub.io.
  - Please correct me and advise if I have things wrong or if I am complicating this more than it is

All, please advise/suggest if there are other thoughts/changes.
We have a history of new contributors also participating in the release process so hoping to simplify as much as possible.

Reply:
agreed with @longwuyuan. To be honest here, this question or change request is out of scope for this PR. At least IMHO. We could consider it but I would prefer to do so in a separate issue.